docs/examples/rails-api/configmap.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-map
+  labels:
+    application: web
+    namespace: some-name
```

Comment:
better to be an explicit name like `rails-api-example`, wonder whether we should use kustomize to layout these examples